src/dra/dra.h

```diff
@@ -703,7 +705,7 @@ extern s16 D_80139A6C;
 extern s16 g_SoundCommandRingBufferWritePos; // D_80139A70
 extern s16 D_80139A74;
 extern s16 D_80139A78;
-extern u_long* D_80139A7C;
+extern u32 D_80139A7C[];
```

Comment:
```
glabel D_80139A7C
/* 99A7C 80139A7C 00000000 */ .word 0x00000000
...
/* 9AE78 8013AE78 00000000 */ .word 0x00000000
.size D_80139A7C, . - D_80139A7C
```
I think this is going to be a `u8 D_80139A7C[0x1400];` given that `99D08 - 99648 + 4` is `0x1400` (the `+ 4` is to consider the next variable starts at `9AE7C`.

I also tried to run the following:
```c
int max = 0;
for (int temp_i = 0; temp_i < 40; temp_i++) {
    int a =  (temp_i & 7) * 8 + (temp_i >> 3) * 0x400;
    if (max < a) {
        max = a;
    }
}
printf("%X", max);
```
and it looks like the maximum is `0x1038`. But let's settle with `0x1400` to play safe and according to how splat disassembled the BSS section.